Libraries/Azure.psm1

```diff
@@ -645,6 +645,15 @@ Function Generate-AzureDeployJSONFile ($RGName, $ImageName, $osVHD, $RGXMLData,
         $DiskType = "Unmanaged"
     }
 
+    $UseSpecializedImage = $CurrentTestData.AdditionalHWConfig.ImageType -contains "Specialized"
+    if ($UseSpecializedImage) {
```

Comment:
Remove this if block. Use $UseSpecializedImage directly at below* code.

Reply:
removed if block